cumulus/parachains/runtimes/testing/rococo-parachain/src/lib.rs

```diff
@@ -653,6 +653,7 @@ pub type SignedExtra = (
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
```

Comment:
Would it not go before the TX charging so that the corrected fee is refunded?

Reply:
No, the fee charging is based on the `PostDispatchInfo` of the extrinsic to apply. This is entirely unaffected by anything the weight reclaim does. We do not downcorrect the fee based on storage reclaim.

Reply:
Okay, so no MEV, i see.